### DISCUSSION TITLE
Using Coral Dual Edge TPU for Frigate LXC and HAOS VM on Proxmox

### DISCUSSION THREAD

**Discussion by hapklaar** [blakeblackshear/frigate#10753] [Detector Support]
I've been using Frigate in a docker container in an LXC container on Proxmox for a while now. I used [this guide](https://www.homeautomationguy.io/blog/running-frigate-on-proxmox) to get everything up and running. The detector is using the GPU - openvino. This works fine most of the time, but with the detector enabled the entire Proxmox host crashes regularly.

I still have a Dual Edge Coral TPU module I don't use, so wanted to try this. To be able to use both TPU's I ordered an adapter to use it in a B/M slot: https://www.makerfabs.com/dual-edge-tpu-adapter-m2-2280-b-m-key.html

I'd like to try two things:

1. Use both TPU's in Frigate
2. Use 1 TPU in Frigate for detection and the other in my HAOS vm, to be able to do some [face recognition with Compreface](https://www.reddit.com/r/frigate_nvr/comments/17nto5u/compreface_docker_with_coral_tpu_support/)

Curious to hear if anyone has gotten either or both options working and can share the steps they took to get there. It would be much appreciated!

**Comment by hapklaar** [comment]
moving to detector support, might be better off there. Hope someone can help!

**Reply by NickM-27** [reply]
None of the maintainers use proxmox so you will need to have a community member that can help.
